Ekinops/ekinops-oneos/ingest/parser.yml

```
@@ -4,13 +4,15 @@ pipeline:
     external:
       name: grok.match
       properties:
-        pattern: "%{CONF_EVNTS}|%{LOG_IN}|%{LOG_OUT}|%{INTERFACE}|%{OTHERS}"
+        pattern: "%{CONF_EVNTS}|%{LOG_IN}|%{LOG_OUT}|%{INTERFACE}|%{TRAFFIC}|%{OTHERS}"
         custom_patterns:
-          CONF_EVNTS: "%{DATA:time} localhost : #%{WORD:log_info}# %{DATA:conf_reason} by (%{DATA:origin}%{SPACE})?(user '%{WORD:user}' from %{DATA:data})? (on|at) %{GREEDYDATA:timestamp}"
-          LOG_IN: "%{DATA:time} localhost : #%{WORD:log_info}# %{DATA:outcome} for (%{DATA:origin}%{SPACE})user '%{WORD:user}' from (Source IP Address '%{DATA:ip}' Local Port '%{WORD:port}'|%{DATA:data}) at %{GREEDYDATA:timestamp}"
-          LOG_OUT: "%{DATA:time} localhost : #%{WORD:log_info}# Logged out user:%{DATA:log_user} from (a|the) %{WORD:origin} session (with client ip: %{DATA:ip}%{SPACE})?on %{GREEDYDATA:timestamp}"
-          INTERFACE: "%{DATA:time} localhost : #%{WORD:log_info}# IF: %{DATA:intarface_name} is (%{DATA:data} |%{NOTSPACE})?%{WORD:reason}"
-          OTHERS: "%{DATA:time} localhost : #%{WORD:log_info}# (%{WORD:provider})? \\[%{WORD:name}\\] %{GREEDYDATA:msg}"
+          PREFIX: "%{DATA:time} localhost : #%{WORD:log_info}#"
+          CONF_EVNTS: "%{PREFIX} %{DATA:conf_reason} by (%{DATA:origin}%{SPACE})?(user '%{WORD:user}' from %{DATA:data})? (on|at) %{GREEDYDATA:timestamp}"
+          LOG_IN: "%{PREFIX} %{DATA:outcome} for (%{DATA:origin}%{SPACE})user '%{WORD:user}' from (Source IP Address '%{DATA:ip}' Local Port '%{WORD:port}'|%{DATA:data}) at %{GREEDYDATA:timestamp}"
+          LOG_OUT: "%{PREFIX} Logged out user:%{DATA:log_user} from (a|the) %{WORD:origin} session (with client ip: %{DATA:ip}%{SPACE})?on %{GREEDYDATA:timestamp}"
+          INTERFACE: "%{PREFIX} IF: %{DATA:interface_name} is (%{DATA:data} |%{NOTSPACE})?%{WORD:reason}"
+          TRAFFIC: "%{PREFIX} list (?P<acl>ACL-WAN-IN|ACL-WAN-OUT) (?P<event_action>permitted|denied) %{WORD:proto} (%{NUMBER:iana_number} )?%{IP:src_ip}:%{NUMBER:src_port}-> %{IP:dst_ip}:%{NUMBER:dst_port} %{NUMBER} matches"
```

According to the `traffic permitted` test, I suggest this change:
```suggestion
          TRAFFIC: "%{PREFIX} list (?P<acl>ACL-WAN-IN|ACL-WAN-OUT) (?P<event_action>permitted|denied) %({WORD:proto}|proto (%{NUMBER:iana_number}) %{IP:src_ip}:%{NUMBER:src_port}-> %{IP:dst_ip}:%{NUMBER:dst_port} %{NUMBER} matches"
```